packages/cli/src/lib/webpack/webpack-base-config.js

```diff
@@ -45,19 +45,6 @@ function resolveTsconfig(cwd, isProd) {
 	}
 }
 
-function getSassConfiguration(...includePaths) {
-	const config = {
-		sourceMap: true,
-		sassOptions: {
-			includePaths,
-		},
-	};
-
-	Object.defineProperty(config, 'includePaths', { value: includePaths });
-
-	return config;
-}
-
```

Comment:
[Exists to smooth over `sass-loader` 7 -> 8 migration](https://github.com/preactjs/preact-cli/pull/930#discussion_r362248387) (current version is v12)